lib/view.js

```diff
@@ -356,7 +356,7 @@ class View extends BaseCommand {
         manifest.dist.integrity && color.yellow(manifest.dist.integrity),
       fileCount:
         manifest.dist.fileCount && color.yellow(manifest.dist.fileCount),
-      unpackedSize: unpackedSize && color.yellow(unpackedSize.value) + ' ' + unpackedSize.unit,
+      unpackedSize: unpackedSize && color.yellow(unpackedSize[0]) + ' ' + unpackedSize[1],
```

Comment:
Do we *really* really care about the unit not being colorized here?

Reply:
That's a weird style choice, tbh.  I'd be fine with just making the whole thing yellow.